modules/vba_documents/lib/vba_documents/pdf_inspector.rb

```diff
@@ -82,13 +83,20 @@ def add_line_of_business(data, parts_metadata)
     def pdf_metadata(pdf)
       metadata = PdfInfo::Metadata.read(pdf)
       dimensions = metadata.page_size_inches
+      if Flipper.enabled?(:vba_documents_larger_page_size_limit)
+        max_height = ::PDFUtilities::PDFValidator::MAX_PAGE_HEIGHT
+        max_width = ::PDFUtilities::PDFValidator::MAX_PAGE_WIDTH
```

Comment:
Similar to my comment above, I think these constants should be defined in the `VBADocuments` module instead since they're specific to the constraints of Benefits Intake's upstream systems.

Reply:
Thanks, this was good advice! I updated this to centralize the default validator options in a static method in `vba_documents` and tried to rely on only those passed-in values for the methods in `lib`. This required me to refactor how error messages are handled in `lib` a little, since some of them depend on the specific validator options in use